htdocs/core/class/commonobject.class.php

```diff
@@ -6005,7 +6005,18 @@ function showOutputField($val, $key, $value, $moreparam='', $keysuffix='', $keyp
 		}
 		elseif ($type == 'array')
 		{
-			$value = implode('<br>', $value);
+			// Handle case of static use (e.g. in lists), where $object not loaded and setVarsFromFetchObj() not called
+			$v = $value;
+			if(! is_array($value)) {
+				$v = @unserialize($value);
```

Comment:
"@unserialize" is hurting me.
Do you have example of content of $value when you need this ?